pytests/noxfile.py

```diff
@@ -13,7 +14,12 @@ def test(session: nox.Session):
     except CommandFailed:
         # No binary wheel for numpy available on this platform
         pass
-    session.run("pytest", *session.posargs)
+    ignored_paths = []
+    if sys.version_info < (3, 10):
+        # Match syntax is only available in Python >= 3.10
+        ignored_paths.append("tests/test_enums_match.py")
```

Comment:
This is where the version-checking magic happens to only run `match`-based tests on Python >= 3.10.

Reply:
Great solution, nicely done